elements/chart/src/main.js

```diff
@@ -52,6 +52,32 @@ export class EOxChart extends LitElement {
     this.unstyled = false;
   }
 
+  _dispatchItemPointerMoveEvent(detail) {
+    /**
+     * Chart item is hovered. Event detail includes
+     * `event` (pointermove event) and `item` (the hovered item on the chart)
+     *
+     */
+    this.dispatchEvent(
+      new CustomEvent("pointermove.item", {
```

Comment:
I'm unsure about the dot in the names of the new events. We've had a discussion here about event name conventions: https://github.com/EOX-A/EOxElements/pull/675#pullrequestreview-1896001234 

E.g. Vue data-table [uses a `event:target` notation](https://v2.vuetifyjs.com/en/api/v-data-table/#events) with colon, i.e. it would be `pointermove:item` here.

Reply:
Good point, I have changed it to the `event:target` format in https://github.com/EOX-A/EOxElements/pull/1511/commits/838586c9874549ee7b07b1c13d113f4986a1b5b0.